src/translations/en.json

```diff
@@ -661,6 +662,8 @@
     "ConnectionDirection": "Connection direction",
     "Top": "Top",
     "Bottom": "Bottom",
+    "DeleteVoltageLevelOnLine": "Delete a voltage level on a line",
+    "ReplacingLine": "Replacing line",
```

Comment:
this new key / label is correct, but in the source code you're using a different key:
<GridSection title="ReplacingLines" />
so the displayed label is wrong "Replacing lines"